.storybook/components/component-info/ComponentInfo.css

```diff
@@ -15,7 +15,7 @@
 
 .o-ui-sb-component-info-title {
     display: inline-block !important;
-    color: var(--o-ui-text-alias-secondary) !important;
+    color: var(--hop-neutral-text-weak) !important;
     font-weight: 550 !important;
     font-size: var(--o-ui-fs-3) !important;
     font-style: normal !important;
```

Comment:
I know they were already there so you can keep the !important, but that is excessive and it triggers me.

Reply:
As it's overriding Storybook, they have their use.